lib/components/Body.js

```diff
@@ -50,6 +56,15 @@ var Body = React.createClass({
     if (isDate(body)) {
       return $(Data, null, body.toString());
     }
+    if (isSet(body)) {
+      var contents = Array.from(body)
```

Comment:
Devcards core aims to be ES5 only, so you can't rely on `Array.from`.

You can use https://www.npmjs.com/package/array.from though.